doc/cprover-manual/unsound_options.md

```diff
@@ -0,0 +1,52 @@
+[CPROVER Manual TOC](../)
+
+# Unsound CLI options for various tools
+
+## Context
+
+In [#6480](https://github.com/diffblue/cbmc/issues/6480), there has been some extensive
+conversation about what it means for certain options to produce *unsound* behaviour.
+
+We concluded that *unsound* in this context is a proxy for the following two behaviours
+that we want to avoid:
+
+* **A spurious counter example to an assertion**, which gives a proof that the assertion
```

Comment:
I think a lot of people would fail to recognize this as a soundness problem. But if you cast it another way - that the tool may report a coverage property (or line of code) as reachable when, in fact, it is not - then I think more people will see that this is just as serious as a spurious proof. I would prefer we state it that way.